toxav/toxav.api.h

```diff
@@ -628,6 +628,18 @@ int toxav_join_av_groupchat(Tox *tox, int32_t friendnumber, const uint8_t *data,
 int toxav_group_send_audio(Tox *tox, int groupnumber, const int16_t *pcm, unsigned int samples, uint8_t channels,
                            unsigned int sample_rate);
 
+
+/**
+ * Sets the Opus bit rate for a group call.
+ *
+ * return 0 on success.
+ * return -1 on failure.
+ */
+
+int toxav_group_bit_rate_set(Tox *tox, int groupnumber, int32_t audio_bit_rate);
+
+bool audio_bit_rate_invalid(uint32_t bit_rate);
```

Comment:
Add documentation. Also, this function should probably test for validity, not for invalidity. Also, it needs a toxav namespace prefix.